doc/thirdpartypackages/index.rst

```diff
@@ -284,3 +284,16 @@ existing table. See the `blume github repository
 
 .. image:: /_static/blume_table_example.png
 
+GUI applications
+****************
+
+sviewgui
+========
+
+`sviewgui <https://pypi.org/project/sviewgui/>`_ is a handy PyQt-based GUI for data visualisation. This GUI is based on the matplotlib and you can visualize your csv file in various ways by only one simple method 'buildGUI()'. Here are the main features;
```

Comment:
- Try to be more to the point in the description. Adjectives like "handy" and "readily" are an opinion and do not add to the information.
- You can use markup to structure the the text, e.g. bullet points.
- Wrap the lines at 79 chars, as that's the convention used elsewhere in the file.

I'd recommend something like this:

~~~
`sviewgui <https://pypi.org/project/sviewgui/>`_ is a PyQt-based GUI for
visualisation of data from csv files or `pandas.DataFrames`. Main features:

- Scatter, line, density, histgram, and box plot types
- Settings for the marker size, line width, number of bins of histgram,
  color map (from cmocean)
- Save figure as editable PDF
- Code of the plotted graph is available so that it can be reused and modified
  outside of sviewgui
~~~